whelk-core/src/main/resources/ext/marcframe.json

```diff
@@ -144,13 +144,13 @@
         "o": "marc:IncompleteAuthorityRecord"
       },
       "HoldingEncLevelType": {
-        "1": "marc:HoldingsLevel1",
-        "2": "marc:HoldingsLevel2",
-        "3": "marc:HoldingsLevel3",
-        "4": "marc:HoldingsLevel4",
-        "5": "marc:HoldingsLevel4WithPieceDesignation",
-        "m": "marc:MixedLevels",
-        "z": "marc:OtherLevel"
+        "1": null,
+        "2": null,
+        "3": null,
+        "4": null,
+        "5": null,
+        "m": null,
+        "z": null
```

Comment:
Did we need these, or could fixedDefault: " " do, so we can remove HoldingEncLevelType entirely from marcframe?

Reply:
I definitely agree that it would be best to remove HoldingEncLevelType completely. If fixedDefault was " " we could easily change to "[17]": null. But unfortunately, fixDefault is "u" which makes it more difficult.